### ISSUE TITLE
cifar10&cifar100 pretrained model

### ISSUE THREAD

**Issue by Wang-Y-S** [google-research/vision_transformer#224]
Hi:

Where can I get the pretrained model on cifar10&cifar100?
like this pic accuracy model
![image](https://user-images.githubusercontent.com/88648670/184658326-9615d487-c3bc-48ae-a973-6b6b99794c41.png)
 
Thank you so much!!
Wang-Y-S

**Comment by andsteing** [comment]
Hi Wang-Y-S

Those numbers are from the private "JFT" models that we cannot open source (because of restrictions about of the internal JFT dataset that was used for the pre-training).

![image](https://user-images.githubusercontent.com/16050001/184697516-6a33b74e-8c9d-4a88-b3fb-6479aa67db54.png)

See Table 3 in https://arxiv.org/pdf/2010.11929.pdf

**Comment by Wang-Y-S** [comment]
Hi:

Thanks for your response!
Where can i get the model pre-training from public dataset(imagenet2012 or imagenet21K)?
I search a lot , but I didn't find any information like paper result.

Thank you so much!!
Wang-Y-S

**Comment by andsteing** [comment]
You can find the ImageNet-21k pre-trained models from the [original ViT paper](https://arxiv.org/pdf/2010.11929.pdf) here. See table 5 for expected results.

https://console.cloud.google.com/storage/browser/vit_models/imagenet21k/

You might also want to look into the "how to train your ViT" paper https://arxiv.org/pdf/2106.10270.pdf that reports significantly better numbers for CIFAR100 in table 5 (numbers for CIFAR10 are not reported). The checkpoints for that paper can be found here:

https://console.cloud.google.com/storage/browser/vit_models/augreg/

That's a lot of checkpoints :) you probably want to take one of the recommended ones here

https://github.com/google-research/vision_transformer#available-vit-models

Also see that README about how to fine-tune for CIFAR10. You might find this Colab particularly useful:

https://colab.research.google.com/github/google-research/vision_transformer/blob/main/vit_jax.ipynb

**Comment by Wang-Y-S** [comment]
Hi:

Thank you so much!!
This is I wanted!!
Let me study~~

Thanks!!
Wang-Y-S
